gratipay/models/team.py

```diff
@@ -158,19 +158,6 @@ def to_dict(self):
         }
 
     def migrate_tips(self):
-        payment_instructions = self.db.all("""
```

Comment:
I think the basic filtering done above should be good enough - removing this check might do more harm than good. 

We generally have 5-10 new teams every week, and ~150 teams in total. Adding the first check alone brings down the number of DB calls from 300 (150 * 2) to 20 (10 * 2). Removing this second check would bring that number down to 10 - but wouldn't give us the flexibility to blindly call `Team.migrate_tips` over the console. Also, the `migrate-tips.py` script isn't covered by our test suite, but `Team.migrate_tips` is. 

Thoughts?